### ISSUE TITLE
Raise an error if you load a blocks with Interface.load

### ISSUE THREAD

**Issue by freddyaboulton** [gradio-app/gradio#2521]
- [x] I have searched to see if a similar issue already exists.

We should raise an error when a developer uses `gr.Interface.load(...)` when the space requested is a blocks app instead of an interface app. Loading a block into an interface will always call the upstream app with `fn_index=0` which is not desired.

So `gr.Interface.load("spaces/user/my-blocks-app")` should raise an error.

**Comment by abidlabs** [comment]
Hmm but `gr.Interface.load()` actually just calls `gr.Blocks.load()` internally. 

Do you have an example of this not working? Maybe the issue is something else?

**Comment by freddyaboulton** [comment]
Yep let's close - was under the impression fn_index was always set to 0 when doing gr.Interface.load which is not the case.
Thank you!
